roles/jenkins/ci-hibernate-org-le-ssl.conf

```diff
@@ -0,0 +1,27 @@
+<IfModule mod_ssl.c>
+<VirtualHost *:443>
+  ServerName ci.hibernate.org
+  ProxyRequests Off
+  AllowEncodedSlashes NoDecode
+  RequestHeader set X-Forwarded-Proto "https"
+
+  # Local reverse proxy authorization override
+  # Most unix distribution deny proxy by default
+  <Proxy http://localhost:8080/*>
+    Order deny,allow
+    Allow from all
+  </Proxy>
+
+  ProxyPreserveHost on
+  ProxyPass / http://localhost:8080/ nocanon
+  ProxyPassReverse / http://localhost:8080/
+SSLCertificateFile /etc/letsencrypt/live/blog.hibernate.org/cert.pem
+SSLCertificateKeyFile /etc/letsencrypt/live/blog.hibernate.org/privkey.pem
```

Comment:
This looks weird (and the line above too). Shouldn't it be `ci.hibernate.org`?

Reply:
no that's the name of the certificate, it has been auto-generated like that; no my favourite choice either but I can no longer change it.